fd-send-recv-tuntap.opam

```diff
@@ -14,10 +14,11 @@ tags: [
   "org:mirage"
```

Comment:
Yes, it's annoying but I thought it does no harm. And I like it more than generating additional stuff in the `jbuild` file if a flag is present (mostly because it requires an additional dependency)